mitodl/ol-django
# Updated request structure and added HMAC signature

Fixes #17
Part of https://github.com/digitalcredentials/sign-and-verify/issues/17

This updates the credential signing API with the following changes:

- Signs request to `sign-and-verify` for credentials with HMAC Signature + Digest headers
- Updates the request body structure to support the new presentation-structured body sent by the wallet app
- Adds a call to perform verification of the learner's presentation 

To test:

- Run [this branch](https://github.com/digitalcredentials/sign-and-verify/pull/18) of the `sign-and-verify` service
- Update `testapp/settings/dev.py`, setting `MITOL_DIGITAL_CREDENTIALS_HMAC_SECRET` to match the shared HMAC secret (any arbitrary string) in the `sign-and-verify` service
- Run the python script + `curl` command it prints in `testapp/README.md`
